dbt_airflow_factory/k8s/k8s_parameters.py

```diff
@@ -123,3 +134,14 @@ def env_vars(self):  # type: ignore
             from kubernetes.client import models as k8s
 
             return [k8s.V1EnvVar(k, v) for k, v in self._env_vars.items()]
+
+    @property
+    def secrets(self) -> Optional[List[Secret]]:
+        """
+        Return list containing secrets to be set in Kubernetes.
+        :return List containing kubernetes Secrets
+        """
+        if self._secrets is None:
+            return None
```

Comment:
Previously, if there were no secrets in config, `secrets` attribute was set to empty array `[]` . After the change, it returns `None`. Isn't this somehow a breaking change?

Reply:
Ok, now I see in the docs, that `None` in this operator is perfectly fine